qa/large-values-network/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.9"
+services:
+
+  # For more information on these configs, refer to:
+  # https://github.com/HathorNetwork/rfcs/blob/master/text/0033-private-network-guide.md
+
+  fullnode:
+    image:
+      ${HATHOR_LIB_INTEGRATION_TESTS_FULLNODE_IMAGE:-hathornetwork/hathor-core}
+    command: [
+      "run_node",
+      "--listen", "tcp:40404",
+      "--status", "8080",
+      "--test-mode-tx-weight",
+      "--wallet-index",
+      "--allow-mining-without-peers",
+      "--unsafe-mode", "testnet-golf",
+      "--memory-storage",
+      "--config-yaml", "privnet/conf/privnet.yml"
+    ]
+    ports:
+      - "8083:8080"
+      - "40404:40404"
+    volumes:
+      - type: bind
+        source: ./
+        target: /privnet/conf
+    networks:
+      - hathor-privnet
+
+  tx-mining-service:
+    image:
+      ${HATHOR_LIB_INTEGRATION_TESTS_TXMINING_IMAGE:-hathornetwork/tx-mining-service}
```

Comment:
@tuliomir for when you run the QA: I couldn't run it like this. I got an error related to CORS and the tx-mining-service in the console:

```
Access to XMLHttpRequest at 'http://localhost:8035/submit-job' from origin 'http://localhost:3000/' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I created a PR in the tx-mining-service that fixes it: https://github.com/HathorNetwork/tx-mining-service/pull/123. However, I'm not sure if I was doing something wrong and whether that PR is really necessary. If it is, please go ahead and review/merge it. Otherwise, you can close it.

If the PR is indeed necessary, you can run the QA without waiting for a new release of the tx-mining-service. Simply checkout to that branch and build a docker image locally with `docker build -t tx-mining-service .`. Then, set `HATHOR_LIB_INTEGRATION_TESTS_TXMINING_IMAGE=tx-mining-service` to override the image used in the docker compose.